docs/user/features/note-templates.md

```diff
@@ -25,27 +25,40 @@ To create a note from a template:
 _Theme: Ayu Light_
 
 ## Special templates
+
 ### Default template
 
 The `.foam/templates/new-note.md` template is special in that it is the template that will be used by the `Foam: Create New Note` command.
-Customize this template to contain content that you want included every time you create a note.
+Customize this template to contain content that you want included every time you create a note. At minimum this template should include the following:
```

Comment:
I wonder if this can be a bit confusing, it can be read a as a template needing to specify a type for the note (which is not the case).
What were you trying to accomplish here?